nav2_smac_planner/src/node_hybrid.cpp

```diff
@@ -401,13 +412,25 @@ void NodeHybrid::resetObstacleHeuristic(
     // must reset values for non-constructed indices
     std::fill_n(
       obstacle_heuristic_lookup_table.begin(), obstacle_size, 0.0);
+    if (motion_table.obstacle_heuristic_admissible) {
+      astar_2d_h_table.resize(size, -1.0f);
+      // must reset values for non-constructed indices
+      std::fill_n(
```

Comment:
This is just a relative euclidean distance cache whose values are not going to be changed with a different goal. However, I'm going to measure whether the whole caching thing has any sense so maybe `astar_2d_h_table` will be deleted.

Reply:
I'm unclear on this - if you later check if the values of `astar_2d_h_table` are `< 0` to know if they're valid to use, you would need to reset them all to be negative between goals. If you had 2 different goal locations on the same map, this would change in meaning to need to be reconstructed, no?